tools/chpl-language-server/src/symbol_signature.py

```diff
@@ -299,3 +303,20 @@ def _intent_to_string(intent: Optional[str]) -> str:
     }
     # use 'intent' as the default, so if no remap no work done
     return remap.get(intent, intent) if intent else ""
+
+def _fncall_to_string(call: chapel.FnCall) -> List[Component]:
+    """
+    Convert a call to a string
+    """
+    comps = []
+
+    comps.extend(_node_to_string(call.called_expression()))
+    comps.append(_wrap_str("[" if call.used_square_brackets() else "("))
+    sep = ""
+    for a in call.actuals():
+        comps.append(_wrap_str(sep))
+        sep = ", "
+        comps.extend(_node_to_string(a))
```

Comment:
This doesn't account for named actuals, and will print incorrectly in the following case:

```Chapel
record myRecord {
  type fst;
  type snd;
}

proc test(x: myRecord(snd = int, fst = bool)) {}
```